src/test/java/org/apache/commons/crypto/stream/AbstractCipherStreamTest.java

```diff
@@ -294,6 +600,26 @@ protected CryptoOutputStream getCryptoOutputStream(
         return new CryptoOutputStream(baos, cipher, bufferSize,
                 new SecretKeySpec(key, "AES"), new IvParameterSpec(iv));
     }
+    
+    protected CryptoOutputStream getCryptoOutputStream(final String transformation,
```

Comment:
can just call the other `getCryptoOutputStream`?

Reply:
Unused constructors were a significant gap in code coverage, so it was an easy win to overload this method and use it to rerun existing tests against them.  Same with the getInputStream, as alluded to in an earlier comment.